src/Xamarin.Android.Build.Tasks/Tasks/RemoveRegisterAttribute.cs

```diff
@@ -24,8 +25,11 @@ public override bool RunTask ()
 		{
 			// Find Mono.Android.dll
 			var mono_android = ShrunkFrameworkAssemblies.First (f => Path.GetFileNameWithoutExtension (f.ItemSpec) == "Mono.Android").ItemSpec;
-
-			using (var assembly = AssemblyDefinition.ReadAssembly (mono_android, new ReaderParameters { ReadWrite = true })) {
+			var path = Path.GetFullPath (Path.GetDirectoryName (mono_android));
+			using var resolver = new DirectoryAssemblyResolver (this.CreateTaskLogger (), loadDebugSymbols: false, loadReaderParameters: new ReaderParameters { ReadWrite = true });
+			resolver.SearchDirectories.Add (path);
```

Comment:
Did they share a project or a repro? I was wondering if a specific NuGet package causes this problem, which would let us add a test.

Reply:
They didn't